### PR TITLE
updated glibc version to 2.24

### PR DESCRIPTION
**Pull request by dennybaa**
This is the most recent version of spark. I want to merge it to master.
@pferrel If you require the previous versions, they will be available [here](https://hub.docker.com/r/actionml/spark/tags/), unless we remove the tags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,6 @@
 
 ## Starting up
 
-First connect to the docker daemon
-
-```
-export DOCKER_HOST=:3375
-```
-
 This repository contains a set of scripts and configuration files to run a [Apache Spark](https://spark.apache.org/) standalone cluster from [Docker](https://www.docker.io/) container.
```
Comment (dennybaa) [inline README.md:7]: @pferrel I've reverted the above ^^.

**Comment by dennybaa** [comment]
@pferrel Any updates?

**Comment by pferrel** [comment]
OK

What is glibc compatibility for? snappy?

**Comment by dennybaa** [comment]
yes, that's the repo you've used to use for TUI if i'm not wrong. alpine-glibc tag ==> 
https://hub.docker.com/r/actionml/spark/tags/

**Pull request merged**